src/mme-app/mmeStates/detachWfDelSessionResp.cpp

```diff
@@ -64,8 +64,13 @@ void DetachWfDelSessionResp::initialize()
         }
         {
                 ActionTable actionTable;
-                actionTable.addAction(&ActionHandlers::process_del_session_resp);
```

Comment:
we have handling for del_Session_resp right ?

Reply:
Yes, If delete session is received, we follow the normal flow, otherwise, send detach accept, delete UE Context.